### ISSUE TITLE
EML is created with extra, malformed allow elements

### ISSUE THREAD

**Issue by kzollove** [EDIorg/hymetDP#15]
From a JRN create_hymetDP.R script

Note the <allow>public</allow> and <allow>read</allow>.

How did they end up there? (Not in the L0 dataset)

```
<access authSystem="https://pasta.edirepository.org/authentication" order="allowFirst" scope="document" system="https://pasta.edirepository.org">
    <allow>public</allow>
    <allow>read</allow>
    <allow>
      <principal>uid=jrn,o=edi,dc=edirepository,dc=org</principal>
      <permission>all</permission>
    </allow>
    <allow>
      <principal>public</principal>
      <permission>read</permission>
    </allow>
  </access>
```

**Comment by gremau** [comment]
I can confirm this when I run the script after updating and fixing CV values. The resulting EML doesn't validate unless I remove the first two <allow> tags.

**Comment by kzollove** [comment]
Resolved in de0fa162bc4a9398bd6cbac252c82b56f5c8202b

The source data package for this script only had a single allow element. `create_eml()` anticipated at least two allow elements, and created malformed EML otherwise. 

Now, `create_eml()` detects single allow elements and handles accordingly.

Thanks for the bug catch, @gremau!
